files/unarchive.py

```diff
@@ -347,7 +347,7 @@ def is_unarchived(self):
             version = pcs[1]
             ostype = pcs[2]
             size = int(pcs[3])
-            path = pcs[7]
+            path = unicode(pcs[7], 'utf-8')
```

Comment:
Don't we test all code against python3 ? I thought if it was approved by Travis/Shippable, things were fine ?